.github/workflows/preview.yml

```
@@ -26,10 +31,10 @@ jobs:
 
       - name: Install and Build
         run: |
-          npm install
+          corepack yarn install --frozen-lockfile
```

`--frozen-lockfile` is for Yarn 1.x, it's been deprecated on modern Yarn (Berry).
```suggestion
          corepack yarn install --immutable
```